### PR TITLE
ITEX sample update for AI Kit product name change

### PR DESCRIPTION
**Pull request by louie-tsai**
# Existing Sample Changes
## Description

updated ITEX sample for AI Kit name changes.

Fixes Issue# ONSAM-1716

## External Dependencies
No

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Implement fixes for ONSAM Jiras

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Command Line
- [ ] oneapi-cli
- [ ] Visual Studio
- [ ] Eclipse IDE
- [ ] VSCode
- [ ] When compiling the compliler flag "-Wall -Wformat-security -Werror=format-security" was used

### CODE REVIEW

**Review by jimmytwei — APPROVED** [review]

File: AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md [hunk 1]
```diff
@@ -1,13 +1,23 @@
 # Intel Extension for TensorFlow Getting Started Sample
-This code sample will guide users how to run a tensorflow inference workload on both GPU and CPU by using oneAPI AI Analytics Toolkit and also analyze the GPU and CPU usage via oneDNN verbose logs
+This code sample will guide users how to run a tensorflow inference workload on both GPU and CPU by using Intel AI Tools and also analyze the GPU and CPU usage via oneDNN verbose logs
```
Comment (avladimi) [inline AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md:2]:
```suggestion
This code sample demonstrates how to run a TensorFlow inference workload on both GPU and CPU by using Intel AI Tools and analyze the GPU and CPU usage via oneDNN verbose logs
```

File: AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
+## Prerequisites
+
+| Optimized for          | Description
+|:---                    |:---
+| OS                     | Ubuntu* 22.04  
+| Hardware               | Intel® Data Center GPU
+| Software               | Intel Extension for TensorFlow
+
+Intel Extension for TensorFlow* is ready for use once you finish the Intel AI Tools installation through AI Tools Selector or Offline Installer.   
+You can refer to the oneAPI AI Tools [product page](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-analytics-toolkit.html) for software installation and the *[Get Started with the Intel® AI Tools for Linux*](https://software.intel.com/en-us/get-started-with-intel-oneapi-linux-get-started-with-the-intel-ai-analytics-toolkit)* for post-installation steps and scripts.
```
Comment (avladimi) [inline AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md:17]:
```suggestion
You can refer to the Intel AI Tools [product page](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-analytics-toolkit.html) for software installation and the *[Get Started with the Intel® AI Tools for Linux*](https://software.intel.com/en-us/get-started-with-intel-oneapi-linux-get-started-with-the-intel-ai-analytics-toolkit)* for post-installation steps and scripts.
```

File: AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md [hunk 3]
```diff
@@ -19,16 +29,13 @@ Code samples are licensed under the MIT license. See
...
 ## Pre-requirements (Local or Remote Host Installation)
 
-TensorFlow* is ready for use once you finish the Intel® AI Analytics Toolkit (AI Kit) installation and have run the post installation script.
+TensorFlow* is ready for use once you finish the Intel AI Tools installation and have run the post installation script.
 
-You can refer to the oneAPI [main page](https://software.intel.com/en-us/oneapi) for toolkit installation and the Toolkit [Intel&reg; AI Analytics Toolkit Get Started Guide for Linux](https://software.intel.com/en-us/get-started-with-intel-oneapi-linux-get-started-with-the-intel-ai-analytics-toolkit) for post-installation steps and scripts.
+You can refer to the oneAPI [main page](https://software.intel.com/en-us/oneapi) for toolkit installation and the Toolkit [Intel&reg; Intel AI Tools Get Started Guide for Linux](https://software.intel.com/en-us/get-started-with-intel-oneapi-linux-get-started-with-the-intel-ai-analytics-toolkit) for post-installation steps and scripts.
 
-## Environment Setup
+## Environment Setup (Only applicable to AI Tools Offline Installer)
```
Comment (avladimi) [inline AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md:38]:
```suggestion
## Environment Setup (Only applicable to Intel AI Tools Offline Installer)
```

File: AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md [hunk 4]
```diff
@@ -1,13 +1,23 @@
 # Intel Extension for TensorFlow Getting Started Sample
-This code sample will guide users how to run a tensorflow inference workload on both GPU and CPU by using oneAPI AI Analytics Toolkit and also analyze the GPU and CPU usage via oneDNN verbose logs
+This code sample will guide users how to run a tensorflow inference workload on both GPU and CPU by using Intel AI Tools and also analyze the GPU and CPU usage via oneDNN verbose logs
 
 ## Purpose
-  - Guide users how to use different conda environments in oneAPI AI Analytics Toolkit to run TensorFlow workloads on both CPU and GPU
+  - Guide users how to use different conda environments in Intel AI Tools to run TensorFlow workloads on both CPU and GPU
   - Guide users how to validate the GPU or CPU usages for TensorFlow workloads on Intel CPU or GPU
```
Comment (avladimi) [inline AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md:6]:
```suggestion
  - Guide users how to validate the GPU or CPU usage for TensorFlow workloads on Intel CPU or GPU
```

File: AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md [hunk 5]
```diff
@@ -1,13 +1,23 @@
...
+
+## Prerequisites
+
+| Optimized for          | Description
+|:---                    |:---
+| OS                     | Ubuntu* 22.04  
+| Hardware               | Intel® Data Center GPU
+| Software               | Intel Extension for TensorFlow
+
+Intel Extension for TensorFlow* is ready for use once you finish the Intel AI Tools installation through AI Tools Selector or Offline Installer.
```
Comment (avladimi) [inline AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md:16]:
```suggestion
Before running the sample, install the Intel Extension for TensorFlow* via the Intel AI Tools Selector or Offline Installer.   
```

File: AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md [hunk 6]
```diff
@@ -1,13 +1,23 @@
 # Intel Extension for TensorFlow Getting Started Sample
-This code sample will guide users how to run a tensorflow inference workload on both GPU and CPU by using oneAPI AI Analytics Toolkit and also analyze the GPU and CPU usage via oneDNN verbose logs
+This code sample will guide users how to run a tensorflow inference workload on both GPU and CPU by using Intel AI Tools and also analyze the GPU and CPU usage via oneDNN verbose logs
 
 ## Purpose
-  - Guide users how to use different conda environments in oneAPI AI Analytics Toolkit to run TensorFlow workloads on both CPU and GPU
+  - Guide users how to use different conda environments in Intel AI Tools to run TensorFlow workloads on both CPU and GPU
```
Comment (avladimi) [inline AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md:5]:
```suggestion
  - Guide on how to use different conda environments in Intel AI Tools to run TensorFlow workloads on both CPU and GPU
```

File: AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md (hunk 4 above)
Comment (avladimi) [inline AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md:6]:
```suggestion
  - Guide on how to validate the GPU or CPU usages for TensorFlow workloads on Intel CPU or GPU
```

File: AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md [hunk 7]
```diff
@@ -19,16 +29,13 @@ Code samples are licensed under the MIT license. See
 
 Third party program Licenses can be found here: [third-party-programs.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/third-party-programs.txt)
 
-## Running Samples on the Intel&reg; DevCloud
-If you are running this sample on the DevCloud, skip the Pre-requirements and go to the [Activate Conda Environment](#activate-conda) section.
-
 ## Pre-requirements (Local or Remote Host Installation)
 
-TensorFlow* is ready for use once you finish the Intel® AI Analytics Toolkit (AI Kit) installation and have run the post installation script.
+TensorFlow* is ready for use once you finish the Intel AI Tools installation and have run the post installation script.
```
Comment (avladimi) [inline AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md:34]:
```suggestion
TensorFlow* is ready for use once you finish the installation and have run the post-installation script.
```

File: AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md [hunk 8]
```diff
@@ -46,9 +53,9 @@ In the end, you will have two new conda environments which are user-tensorflow-g
 ## How to Build and Run
 
 You can run the Jupyter notebook with the sample code on your local
-server or download the sample code from the notebook as a Python file and run it locally or on the Intel DevCloud.
+server.
+
 
-**Note:** You can run this sample on the Intel DevCloud using the Dask and OmniSci engine backends for Modin. To learn how to set the engine backend for Intel Distribution of Modin, visit the [Intel® Distribution of Modin Getting Started Guide](https://www.intel.com/content/www/us/en/developer/articles/technical/intel-distribution-of-modin-getting-started-guide.html). The Ray backend cannot be used on Intel DevCloud at this time. Thank you for your patience.
 
 ### Run the Sample in Jupyter Notebook<a name="run-as-jupyter-notebook"></a>
```
Comment (Marina-Zubova) [inline AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md:61]: At the end of the file we need to insert:
*Other names and brands may be claimed as the property of others. [Trademarks](http://www.intel.com/content/www/us/en/legal/trademarks.html)

This request is relevant for any README file where we use asterisks

File: AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md [hunk 9]
```diff
@@ -19,16 +29,13 @@ Code samples are licensed under the MIT license. See
 
 Third party program Licenses can be found here: [third-party-programs.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/third-party-programs.txt)
 
-## Running Samples on the Intel&reg; DevCloud
-If you are running this sample on the DevCloud, skip the Pre-requirements and go to the [Activate Conda Environment](#activate-conda) section.
-
 ## Pre-requirements (Local or Remote Host Installation)
```
Comment (Marina-Zubova) [inline AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md:32]:
```suggestion
## Pre-requisites (Local or Remote Host Installation)
```

**Review by Marina-Zubova — CHANGES_REQUESTED** [review]
This document also needs language and grammar check

File: AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md (hunk 8 above)
Comment (louie-tsai) [reply AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_TensorFlow_GettingStarted/README.md:61]: addressed it accordingly

**Review by ishaghosh27 — APPROVED** [review]

**Pull request merged**